pkg/s3select/input.go

```diff
@@ -109,13 +251,46 @@ type Options struct {
 	// Size of incoming object
 	StreamSize int64
 
+	// OutputSerializationValue
```

Comment:
Do you plan to add something for later

Reply:
OutputSerializationValue will be used to tell the Output type i.e. csv / Json

Reply:
meaning that the file can be `CSV` or `JSON` and we will still allow the output to be of certain type? does S3 allow this behavior?

Reply:
S3 docs says ```The format of the InputSerialization doesn't need to match the format of the OutputSerialization. So, for example, you can specify JSON in the InputSerialization and CSV in the OutputSerialization.```